src/Components/Landing/ArticleLanding.jsx

```diff
@@ -4,6 +4,7 @@ import { Header } from "../Header"
 import * as API from '../../Api'
 import { useParams } from 'react-router-dom'
 import { useState } from "react"
+import { Voter } from "../Voter"
```

Comment:
Why are you bringing in the voter in this component as you dont appear to be using it